x-pack/platform/plugins/shared/alerting/server/saved_objects/rule_template_mappings.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { pick } from 'lodash';
+import type { SavedObjectsTypeMappingDefinition } from '@kbn/core/server';
+
+import { alertMappings } from '../../common/saved_objects/rules/mappings';
+
+export const ruleTemplateMappings: SavedObjectsTypeMappingDefinition = {
+  dynamic: false,
+  properties: {
+    ...pick(alertMappings.properties, ['name', 'tags', 'consumer']),
```

Comment:
Only included fields here that felt relevant for indexing, and have potential search use cases.

Reply:
nit: I'm not sure how kosher it is to use another SO type's mappings as yours.  It will obviously work today, but I'm wondering if it will cause problems in the future.